pkgs/applications/video/mpv/default.nix

```diff
@@ -105,6 +105,11 @@ in stdenv.mkDerivation rec {
     sha256 = "138921kx8g6qprim558xin09xximjhsj9ss8b71ifg2m6kclym8m";
   };
 
```

Comment:
Can you add a comment with the upstream pr and why we need the patch?

Reply:
Done.